### DISCUSSION TITLE
Suggestions for changes for website

### DISCUSSION THREAD

**Discussion by serhii-londar** [serhii-londar/open-source-mac-os-apps#631] [Ideas]
Hey , i apologise if my PR doesn't help you out as you need . I just started to learn coding and using Github or any OS platform for that matter but 
 i have a few ideas you could change for your website .

 1) Main Page - You should use the main page as a way of telling the user who you are , what this website is used for and where do you think
                this project will go ( like your goal for this project ) .

 2) Tabs - I was thinking if you could add tabs in the orange space at the top of the website with all of the categories such as :

           Join Us | Development | Downloader | Editors | Utilities | Support Us | 

           as this will reduce the amount of items on the main page and keep everything in order and easier accessibility .

 3) Description -  You could give the user a bit of information of what is what , say on the main page you've given a list of languages 
                   used to develop apps on MacOs . You could give a bit of information of each language , it's creators and examples of apps
                   developed using these languages .

 4) Logo - You can add the logo on the top orange bar as if the user navigates through your website using the tabs , they can come to the 
           main webpage by clicking on your logo .

 These are my suggestions for your website , i hope this helps and i apologise for not contributing via coding as i've just started to learn c++ , 
 but i hope this helps and all the best for your project.

**Comment by priynshuchouhn** [comment]
hey I am new to github and want to contribute in your project. is there anything I can do for you
